packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -66,6 +67,49 @@ const VersionsTab = (props: VersionsTabProps) => {
     }
   }
 
+  const [openMenuId, setOpenMenuId] = useState(null);
+
+  const toggleContextMenu = useCallback((rowId) => {
+    setOpenMenuId((currentOpenMenuId) =>
+      currentOpenMenuId === rowId ? null : rowId
+    );
+  }, []);
+
+  const closeContextMenu = () => {
+    setOpenMenuId(null);
+  };
+
+  const onDownloadClick = useCallback(
+    async (changesetIndex: number) => {
+      toaster.closeAll();
+      try {
+        const checkpointInfo = await changesetClient.getChangesetCheckpoint(
+          imodelId,
+          changesetIndex
+        );
+        const downloadUrl = checkpointInfo._links.download.href;
+        window.open(downloadUrl, "_blank");
```

Comment:
have you tried using `anchor` as menu item and download there...?

Reply:
otherwise, you might need this for security, I think..
`rel="noopener noreferrer"`

Reply:
@veekeys Programatically provided the "noopener,noreferrer"